pvlib/losses.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""
+This module contains functions for losses of various types: soiling, mismatch,
+snow cover, etc.
+"""
+
+import numpy as np
+import pandas as pd
+from scipy.special import erf
```

Comment:
Let's use numpy.erf instead of scipy.special.erf. That's the cause of the test failure for the py35min environment.